rc/lsp.kak

```diff
@@ -1415,6 +1423,49 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-indent-guides-request -docstring "lsp-indent-guides-request: request updating lsp_indent_guides for the buffer" %{
```

Comment:
as a convention, we only add the command to the docstring if there are any arguments

```suggestion
define-command -hidden lsp-indent-guides-request -docstring "request updating lsp_indent_guides for the buffer" %{
```